Resources/config/fieldtypes.yml

```diff
@@ -1,16 +1,12 @@
-parameters:
-    ezsystems.tweetbundle.fieldType.eztweet.class: EzSystems\TweetFieldTypeBundle\eZ\Publish\FieldType\Tweet\Type
-    ezsystems.tweetbundle.fieldType.eztweet.converter.class: EzSystems\TweetFieldTypeBundle\eZ\Publish\FieldType\Tweet\LegacyConverter
-
 services:
-    ezsystems.tweetbundle.fieldType.eztweet:
+    ezsystems.tweetbundle.fieldtype.eztweet:
         parent: ezpublish.fieldType
-        class: %ezsystems.tweetbundle.fieldType.eztweet.class%
+        class: 'EzSystems\TweetFieldTypeBundle\eZ\Publish\FieldType\Tweet\Type'
```

Comment:
Unless I'm mistaken, the quotes aren't necessary (see the examples from http://symfony.com/doc/current/service_container.html). I'd remove them altogether.